theseus/extlib/tests/test_baspacho_simple.py

```diff
@@ -121,14 +112,11 @@ def check_simple(verbose=False, dev="cpu"):
     assert all(np.linalg.norm(res) < 1e-10 for res in residuals)
 
 
-@pytest.mark.baspacho
-@requires_baspacho
+@run_if_baspacho()
 def test_simple_cpu():
     check_simple(dev="cpu")
 
 
-@pytest.mark.cudaext
```

Comment:
found! restore the cudaext mask and CI will succeed ;)

Reply:
Added, thanks!